apps/system/utils/utils_ps.c

```diff
@@ -98,12 +98,19 @@ static const char *utils_ttypenames[4] = {
 	"--?--  "
 };
 
+#ifdef CONFIG_SMP
+uint32_t cpu;
```

Comment:
Global variable is necessary?
If yes, how about static?

Reply:
Changed to static.